db/blip_handler.go

```diff
@@ -385,8 +401,9 @@ func (bh *blipHandler) sendChanges(sender *blip.Sender, opts *sendChangesOptions
 
 	// Create a distinct database instance for changes, to avoid races between reloadUser invocation in changes.go
 	// and BlipSyncContext user access.
-	changesDb := bh.copyContextDatabase()
-	_, forceClose := generateBlipSyncChanges(bh.loggingCtx, changesDb, channelSet, options, opts.docIDs, func(changes []*ChangeEntry) error {
+	bh.updateUser()
```

Comment:
I don't think we want to update the user on the handler here - the idea is that we're creating a new DB instance unrelated to the handler that we're using for changes.

Reply:
If you agree, I suggest moving the contents of updateUser back into refreshUser, since it would be the only usage.